go.mod

```diff
@@ -215,13 +214,14 @@ require (
 	github.com/otiai10/copy v1.12.0
 	github.com/pierrec/lz4/v4 v4.1.18
 	github.com/pkg/xattr v0.4.9
+	github.com/prometheus/prometheus v0.54.1
 	github.com/shirou/gopsutil/v3 v3.22.10
 	github.com/tklauser/go-sysconf v0.3.10
 	github.com/xdg-go/scram v1.1.2
 	go.elastic.co/apm/module/apmelasticsearch/v2 v2.6.0
 	go.elastic.co/apm/module/apmhttp/v2 v2.6.0
 	go.elastic.co/apm/v2 v2.6.0
-	go.mongodb.org/mongo-driver v1.5.1
+	go.mongodb.org/mongo-driver v1.14.0
```

Comment:
Can't this cause problems for the mongodb module? Maybe we should only bump prometheus and related dependencies?

Reply:
Being its not a major version bump it shouldn't. I only did `go get -u github.com/prometheus/prometheus@v0.54.1`, and it updated all these deps.

I am really surprised these deps are not being automatically upgraded with dependabot.